lib/sections/Landing/index.js

```diff
@@ -312,16 +316,18 @@ var Landing = Vue.extend({
 
   //    select("#matrix").style.margin=  Math.max(0, (window.scrollY - 125))+"px 0 0 0";
    
-
-      if(window.scrollY>=100){
-       select("#matrix").setAttribute("class", "fixed");
-       select("#matrix").style.left = Math.round(matrixLeft)+"px";
-       select("#matrix").style.width = Math.round(matrixWidth)+"px";
-       
-      }else {
-        select("#matrix").setAttribute("class", "");
-        matrixLeft = select("#matrix").getBoundingClientRect().left; 
+      if (settings.isDesktop) {
```

Comment:
@maximobelen We should be doing the isDesktop check at the function call (line 44) and not in the function itself.